build/pmm-client-docker/Dockerfile

```diff
@@ -1,20 +1,25 @@
-FROM centos:7
+FROM registry.access.redhat.com/ubi7/ubi
 RUN yum -y install iproute nc \
```

Comment:
Do we use them? If not, let's remove them.

Reply:
done